recipes/bayestyper/meta.yaml

```diff
@@ -0,0 +1,31 @@
+{% set version = "1.5" %}
+
+package:
+  name: bayestyper
+  version: {{ version }}
+
+build:
+  number: 0
+  skip: True # [osx]
+
+source:
+  url: https://github.com/bioinformatics-centre/BayesTyper/archive/v{{ version }}.tar.gz
+  sha256: 917cd1b1ca7b5cfb6b8327138515ca1ad70878f0c4e8df393fddcbe42f281e14
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - boost
+  host:
+    - zlib
```

Comment:
Does this need zlib during the run time too or is it just statically compiling it in? In the latter case you can just put zlib in the build section.

Reply:
No idea. I have tried removing it. I know very little (that is, nothing) about C++. Just noticed that `zlib` always seems to be a requirement.